testing/internal/e2e/infra/docker.go

```diff
@@ -239,7 +239,7 @@ func StartOpenSshServer(t testing.TB, pool *dockertest.Pool, network *dockertest
 
 	networkAlias := "target"
 	resource, err := pool.RunWithOptions(&dockertest.RunOptions{
-		Repository: "lscr.io/linuxserver/openssh-server",
+		Repository: "docker.mirror.hashicorp.services/linuxserver/openssh-server",
```

Comment:
Updating it here in the test for now. Will look at potentially refactoring this with the other docker work I have ongoing (using docker containers to run more of the e2e tests).